src/backend/chat.nim

```diff
@@ -75,15 +75,18 @@ proc sendChatMessage*(
     }
   ])
 
-proc sendImages*(chatId: string, images: var seq[string]): RpcResponse[JsonNode] {.raises: [Exception].} =
+proc sendImages*(chatId: string, images: var seq[string], msg: string): RpcResponse[JsonNode] {.raises: [Exception].} =
+  # Add "placeholder" text to instead of empty string as mobile app
+  let text = if (msg == ""): "placeholder"  else: msg
+  
   let imagesJson = %* images.map(image => %*
       {
         "chatId": chatId,
         "contentType": 7, # TODO how do we unhardcode this
         "imagePath": image,
         # TODO is this still needed
         # "ensName": preferredUsername,
-        "text": "🖼️"
+        "text": text
```

Comment:
The emoji is imho still used in notifications; now the `placeholder` text will be visible there